src/components/styles/NavBar.css

```diff
@@ -0,0 +1,61 @@
+.navBar {
+  background: linear-gradient(#72107c, #292761);
+  top: 0;
+  left: 0;
+  margin: auto;
+  width: 100%;
+  height: 80px;
+
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.parent {
+  top: 0;
+  left: 0;
+  margin: auto;
+  width: 100%;
+  height: 80px;
+}
+
+.headernav {
+  display: flex;
+  background: linear-gradient(#72107c, #292761);
+  justify-content: flex-start;
+
+  padding: 0px 90px;
+}
+
+.Player {
+  height: 360px;
+  width: 440px;
```

Comment:
You can leave your styles like that but in the future I suggest you use pixels only for font sizes, and for actual component sizes, use relative unit measures like `em`. As your application starts to grow, it's easier to write responsive styles with relative units. I'll leave you an interesting article about it: https://thecssworkshop.com/lessons/relative-units